SPECS-SIGNED/systemd-boot-signed/systemd-boot-signed.spec

```diff
@@ -0,0 +1,91 @@
+%global debug_package %{nil}
+%ifarch x86_64
+%global buildarch x86_64
+%endif
+
+# Support for quick builds with rpmbuild --build-in-place.
+# See README.build-in-place
+%bcond inplace 0
+Summary:        Signed systemd-boot for %{buildarch} systems
+Name:           systemd-boot-signed-%{buildarch}
```

Comment:
Why include `buildarch` in the name?

Reply:
I was following how other specs are written:
- https://github.com/microsoft/azurelinux/blob/2.0/SPECS-SIGNED/grub2-efi-binary-signed/grub2-efi-binary-signed.spec
- https://github.com/microsoft/azurelinux/blob/2.0/SPECS-SIGNED/kernel-signed/kernel-signed.spec

Reply:
Huh, interesting. Thanks.